mule-artifact-it/mule-deployer-it/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.mule.tools.maven</groupId>
         <artifactId>mule-artifact-it</artifactId>
-        <version>3.5.0-SNAPSHOT</version>
+        <version>3.5.0-SANITIES</version>
```

Comment:
Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.